.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,21 +4,41 @@ about: Create a report to help us improve
 labels: bug
 ---
 
-**Describe your environment.** Describe any aspect of your environment relevant to the problem:
-* SDK version: (nuget versions of the all the relevant packages)
-* .NET runtime version (.NET or .NET Core, TargetFramework in the .csproj file): 
-* Platform and OS version:
+# Bug Report
 
-If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on master.
+List of [NuGet packages](https://www.nuget.org/profiles/OpenTelemetry) and
+version that you are using:
 
-**Steps to reproduce.**
-Describe exactly how to reproduce the error. Include a code sample if applicable.
+* `OpenTelemetry 0.4.0-beta.2`
```

Comment:
did you meant this as an example?

Reply:
Yes, do you think we should leave it as blank?

Reply:
Can you say something like (eg: OpenTelemetry 0.4.0-beta2) to make it obvious that its an example

Reply:
Good idea.